src/Util/TestListener.php

```diff
@@ -11,15 +11,22 @@
 
 namespace Symfony\Polyfill\Util;
 
+use PHPUnit\Framework\AssertionFailedError;
+use PHPUnit\Framework\Warning;
+use PHPUnit\Framework\SkippedTestError;
+use PHPUnit\Framework\Test;
+use PHPUnit\Framework\TestListener as PHPUnit_Framework_TestListener;
+use PHPUnit\Framework\TestSuite;
+
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-class TestListener extends \PHPUnit_Framework_TestSuite implements \PHPUnit_Framework_TestListener
+class TestListener extends TestSuite implements PHPUnit_Framework_TestListener
```

Comment:
This test listener is now totally incompatible with older versions of PHPUnit. This is not good (Symfony itself relies on support for older versions due to PHP support).